### ISSUE TITLE
[A11y] View playlist page - Ramp components

### ISSUE THREAD

**Issue by Dananji** [samvera-labs/ramp#833]
### Description

From the accessibility checks performed on playlist related pages in https://github.com/avalonmediasystem/avalon/issues/6312, the following were identified for Ramp components in the view playlist page.
| Success criterion | WCAG 2.1 Level | Accessibility checker | Error | Relevant HTML element |
| --- | --- | --- | --- | --- |
| 2.5.3 Label in Name | Level A | SiteImprove | Visible label and accessible name do not match | Playlist item links in `StructuredNavigation` component in Ramp |
| 4.1.2 Name, Role, Value | Level A | SiteImprove & WAVE  | Form field missing a label | `<input>` textbox for both `Name` and `Time` in edit marker form |

### Done Looks Like
- [ ] Structure items should have identical visible label and accessible names
- [ ] Each form field in the marker edit forms are associated with an accessible label
